### PR TITLE
Support for https endpoints

### PR DESCRIPTION
**Pull request by rodolfomiranda**
This PR replace PR #91 due branch inconsistencies after moving from main to development.

### CODE REVIEW

File: src/keria/app/agenting.py [hunk 1]
```diff
@@ -889,31 +888,34 @@ def on_get(req, rep, alias):
         if role in (kering.Roles.witness,):  # Fetch URL OOBIs for all witnesses
             oobis = []
             for wit in hab.kever.wits:
-                urls = hab.fetchUrls(eid=wit, scheme=kering.Schemes.http)
+                urls = hab.fetchUrls(eid=wit, scheme=kering.Schemes.http) or hab.fetchUrls(eid=wit, scheme=kering.Schemes.https)
```
Comment (m00sey) [inline src/keria/app/agenting.py:891]: should add a test to cover this branch in logic

**Review by m00sey — CHANGES_REQUESTED** [review]
tests

**Comment by kentbull** [comment]
The main difference between these two PRs was the usage of `urljoin` to combine the parsed URL `up` with the rest of the OOBI URL string. I added that to #103 and we can merge that.

File: src/keria/app/agenting.py (hunk 1 above)
Comment (kentbull) [reply src/keria/app/agenting.py:891]: Agreed. We don't have coverage of that branch. In order to fully test that branch it seems we need to bring in locally self signed certificates for proper TLS. I could bring that in from HIO like I mentioned. Would you still prefer mocking in this case? We'd just need a hab with TLS urls. I'm not familiar with the test fixtures though I suppose there is a Hab test fixture pretty close to what we need.

Comment (kentbull) [reply src/keria/app/agenting.py:891]: Yeah, looks like we'll need a new `helpers.openKeria()` test fixture that returns an agency started up with TLS config. Want me to make that for this PR?

**Comment by rodolfomiranda** [comment]
I'm closing this PR since all the changes were included in PR #102

**Comment by rodolfomiranda** [comment]
I'm closing this PR since all the changes were included in PR #102

**Pull request closed without merging**